domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -134,7 +132,18 @@ constructor(
             )
             settingsRepository.updateVideoStabilizationSupported(isStabilizationSupported())
         }
-        videoCaptureUseCase = createVideoUseCase()
+
+        videoCaptureUseCase = when (currentCameraSettings.targetFrameRate) {
+            TargetFrameRate.TARGET_FPS_NONE -> {
+                VideoCapture.withOutput(recorder)
+            }
+
+            else -> {
+                VideoCapture.Builder(recorder)
+                    .setTargetFrameRate(currentCameraSettings.targetFrameRate.range)
```

Comment:
It looks like this is only happening once during initialization. Shouldn't we be reacting to changes in this frame rate and updating the use cases / rebinding?

Reply:
before the update introducing perpetual settings, the settings would automatically be applied whenever navigating back to the preview screen.
like i mentioned offline, it now has the same issue as #131 where the setting is only applied on app startup, so I will need to still to take care of that